src/app/(nextjs_migration)/(authenticated)/user/breaches/page.tsx

```diff
@@ -150,6 +150,35 @@ export default async function UserBreaches() {
     return breachRowsHTML;
   }
 
+  // Fetch list of subscriptions.
+  const result = await fetch(
+    `${process.env.OAUTH_API_URI}/oauth/mozilla-subscriptions/customer/billing-and-subscriptions`,
+    {
+      headers: {
+        "Content-Type": "application/json",
+        "authorization": `Bearer ${session?.user.subscriber?.fxa_access_token}`,
```

Comment:
Yes, per the typescript definition for `session`... I think in practice we should always have the `fxa_access_token` if the user is logged in, but I can check this and throw earlier just in case.